src/App.js

```diff
@@ -1,6 +1,9 @@
 import "./index.css";
 import headerLogo from "./images/logo_place_header.svg";
+import footerLogo from "./images/logo_place_footer.svg";
 import headerImage from "./images/two_again.png";
+import feynman from "./images/feynman.png";
```

Comment:
1. This should be a background image, not an element. 
Check [background](https://www.w3schools.com/cssref/css3_pr_background.asp) property in docs. Also, multiple backgrounds can help you to figure it out (you do not need multiple here, but an understanding of the concept would give you a hint). If it would be too complex, split it into background-position, background-size, and background-image.

2. Since we do not need this image here it is fine to remove it, but the naming is not nice. `feynman` means do not explain what is that. How about `feynmanBackgroundImage` or `feynmanImage` for the future?

Reply:
done